public/locales/en/translation.json

```
@@ -1865,6 +1896,8 @@
     "mostHoursVolunteered": "Most Hours Volunteered",
     "leastHoursVolunteered": "Least Hours Volunteered",
     "accepted": "Accepted",
+    "groupTable": "group table",
```

Biome (2.1.2)

[error] The key groupTable was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)